lib/cext/ABI_check.txt

```diff
@@ -1 +1 @@
-3
+4
```

Comment:
Technically a C extension's extconf.rb might do different things if some of these RbConfig keys are set so it would be safer to bump https://github.com/oracle/truffleruby/blob/master/lib/cext/include/truffleruby/truffleruby-abi-version.h instead